cluster/bounded_log_adapter.go

```diff
@@ -111,6 +125,9 @@ func (adapter *boundedLogAdapter) onMessage(
 			string(event.ResponseChannel),
 			event.EncodedPrincipal,
 		)
+		if err != nil {
```

Comment:
probably panic here, things are unlikely to work if this errors

Reply:
Done